Networking/NetworkingTests/Mapper/CountryListMapperTests.swift

```diff
@@ -43,23 +37,23 @@ private extension CountryListMapperTests {
 
     /// Returns the CountryListMapperTests output upon receiving `filename` (Data Encoded)
     ///
-    func mapCountries(from filename: String) -> [Country]? {
+    func mapCountries(from filename: String) throws -> [Country]? {
```

Comment:
Not directly related to this PR, but since the method is now throwing an exception, what about throwing an exception in the `guard`-`else`, then making it return a non-nil `[Country]` and propagating this change to other call sites?

The alternative is to stop throwing these exceptions and just `try?` instead of `try!`, but consequently missing them -- which for this test I suppose can be helpful.